src/resources/armies/borgo.json

```diff
@@ -60,10 +65,151 @@
       ],
       "modifiers": [
         {
-          "type": "initiative",
-          "value": -128
+          "type": "net"
         }
       ]
+    }, {
+      "id": "mutant",
+      "initiative": 2,
+      "health": 2,
+      "count": 6,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
+        }, {
+          "type": "melee",
+          "angle": 5
+        }, {
+          "type": "melee",
+          "angle": 1
+        }
+      ]
+    }, {
+      "id": "butcher",
+      "initiative": 2,
+      "health": 2,
+      "count": 4,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 2
+        }, {
+          "type": "melee",
+          "angle": 5
+        }
+      ]
+    }, {    
+      "id": "super_mutant",
+      "initiative": 2,
+      "health": 1,
+      "count": 1,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 2
+        }
+      ]
+    }, {
+      "id": "super_officer",
+      "count": 1,
+      "health": 1,
+      "module": true,
+      "modifiers": [
+	{
+	  "type": "strengthm",
+	  "value": 1
+	}, {
+	  "type": "strengthm",
+	  "angle": 1,
+	  "value": 1
+	}, {
+	  "type": "strengthm",
+	  "angle": 5,
+	  "value": 1
+	}, {
+	  "type": "toughness",
```

Comment:
I don't think super officer has such modifier.

Reply:
It is described like this in player help sheet but its the same as extra health point, modifier removed.